api/src/main/scala/hmda/api/http/HttpApi.scala

```diff
@@ -21,12 +21,16 @@ trait HttpApi extends HmdaApiProtocol {
   val rootPath =
     pathSingleSlash {
       get {
-        complete {
-          val now = Instant.now.toString
-          val host = InetAddress.getLocalHost.getHostName
-          val status = Status("OK", "hmda-api", now, host)
-          log.debug(status.toJson.toString)
-          ToResponseMarshallable(status)
+        time {
+          val requestTime = System.currentTimeMillis()
+          complete {
+            val now = Instant.now.toString
+            val host = InetAddress.getLocalHost.getHostName
+            val status = Status("OK", "hmda-api", now, host)
+            log.debug(status.toJson.toString)
+            log.debug("Elapsed time: " + (System.currentTimeMillis() - requestTime) + "ms")
```

Comment:
It looks like this line is a leftover from previous implementation? If that is the case, `requestTime` above should also be removed